sys/src/9/port/devether.c

```diff
@@ -20,11 +20,14 @@
 
 static Ether *etherxx[MaxEther];
 
+extern Ether *archetherprobe(int ctlrno, char *type, int (*reset)(Ether *));
+extern void archethershutdown(Ether *ether);
+
 Chan*
 etherattach(char* spec)
 {
 	Proc *up = externup();
-	uint32_t ctlrno;
+	int ctlrno;
```

Comment:
Btw: if it were global or meant to be persistent I'd generally agree; but it's not.